test_unstructured/documents/structured_jsons/example.json

```diff
@@ -0,0 +1,121 @@
+[
+    {
+        "element_id": "897a8a47-377c-4ad6-aab8-39a929879537",
+        "metadata": {
+            "category_depth": 0,
+            "text_as_html": "<body class=\"Document\" id=\"897a8a47-377c-4ad6-aab8-39a929879537\" />"
```

Comment:
we should not include `class="Document"`'s in Unstructured Elements output.
by convention, a list of Unstructured elements represents a document, so this is not adding any information (and gets in the way if combining multiple pages

Reply:
more detail: this may seem nit-picky, but the reason i feel strongly we should drop `Document` is it violates the standard pattern for Unstructured output for all other documents that are partitioned with `partition()`, i.e. the result is a set of Unstructured Elements that immediately begins with document content - there are no elements reserved for the Document itself.

in the past, we have discussed introducing something like a `DocumentMeta` element `type` in the past, and that could still be on the table.

all of that said, i don't think this strictly needs to be a blocker at this point.